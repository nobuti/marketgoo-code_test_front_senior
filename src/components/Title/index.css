.Title {
    margin-bottom: var(--size-6);
}
