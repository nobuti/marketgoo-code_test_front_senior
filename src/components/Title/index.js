import React from 'react'
import PropTypes from 'prop-types'

import './index.css'

const Title = ({ children }) => <h1 data-testid="h1" className="ola-title Title">{children}</h1>

Title.displayName = 'Title'

Title.propTypes = {
  children: PropTypes.node
}

export default Title
