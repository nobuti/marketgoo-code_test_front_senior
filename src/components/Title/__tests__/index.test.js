import React from 'react'
import { render, screen } from '@testing-library/react'
import Title from '../'

describe('Title', () => {
  test('renders properly', () => {
    const { container } = render(
      <Title>Wadus</Title>
    )

    expect(container).toMatchSnapshot()
    expect(screen.queryByTestId('h1')).toHaveClass('ola-title Title')
  })
})
