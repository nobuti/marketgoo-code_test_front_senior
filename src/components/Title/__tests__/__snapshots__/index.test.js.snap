// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders properly 1`] = `
<div>
  <h1
    class="ola-title Title"
    data-testid="h1"
  >
    Wadus
  </h1>
</div>
`;
